app/models/spree/order.rb

```diff
@@ -517,6 +465,21 @@ def create_proposed_shipments
       shipments
     end
 
+    # Clear shipments and move order back to address state
```

Comment:
there's an if missing in this comment:
"Clear shipments and move order back to address state if order is not complete but contains shipments" ?